push_notifications/models.py

```diff
@@ -98,7 +102,7 @@ class GCMDevice(Device):
 	registration_id = models.TextField(verbose_name=_("Registration ID"), unique=SETTINGS["UNIQUE_REG_ID"])
 	cloud_message_type = models.CharField(
 		verbose_name=_("Cloud Message Type"), max_length=3,
-		choices=CLOUD_MESSAGE_TYPES, default="FCM",
+		choices=CLOUD_MESSAGE_TYPES, default=get_default_cloud_message_type(),
```

Comment:
```suggestion
		choices=CLOUD_MESSAGE_TYPES, default=get_default_cloud_message_type,
```

having the callable method as default makes it easier to override the setting in tests. also requires the regeneration of the migration